big_tests/tests/connect_SUITE.erl

```diff
@@ -207,6 +209,10 @@ end_per_testcase(replaced_session_cannot_terminate_different_nodes = CaseName, C
     distributed_helper:remove_node_from_cluster(mim2(), Config),
     mongoose_helper:restore_config(Config),
     escalus:end_per_testcase(CaseName, Config);
+end_per_testcase(verify_peer_disconnects_when_client_has_no_cert, Config) ->
+    mongoose_helper:restore_config(Config),
+    catch escalus_event:stop(Config),
+    catch escalus_cleaner:stop(Config);
```

Comment:
Just a note: Alternatively I think that we could just have an `after` clause in the `try` expression in the test itself. It could call `escalus_connection:kill`. But this version is also OK for me.